exporter/keystoneexporter/keystone.go

```diff
@@ -196,7 +196,7 @@ func publishMessage(b []byte) error {
 	}
 
 	if httpResponse.StatusCode == http.StatusOK && err == nil {
-		log.Debugf("published keystone message with size: %d", len(b))
+		log.Infof("published keystone message with size: %d", len(b))
```

Comment:
does this really want to be info? it's pretty busy already.

Reply:
No.  That's a mistake, thanks.